packages/kamino-sdk/src/Kamino.ts

```diff
@@ -5585,7 +5585,7 @@ export class Kamino {
 
     const decimalsA = await getMintDecimals(this._connection, whilrpoolState.tokenMintA);
     const decimalsB = await getMintDecimals(this._connection, whilrpoolState.tokenMintB);
-    const tickIndex = priceToTickIndex(price, decimalsA, decimalsB);
+    const tickIndex = getNextValidTickIndex(priceToTickIndex(price, decimalsA, decimalsB), whilrpoolState.tickSpacing);
```

Comment:
@mihalex98 I think this need to follow this logic - https://github.com/hubbleprotocol/hubble-common/blob/53b8eee17548c975d25bc1e749607387cdf07b76/packages/kamino-sdk/src/Kamino.ts#L3385

Reply:
Hmm, this can indeed cause issues when the tick is already initialized, and it should not be a breaking change. @silviutroscot do you agree?

Reply:
yeah, I think it is fine to call `getNextValidTickIndex` to make sure it is the correct tick